### PR TITLE
doc: update the description for SPDK in bluestore-config-ref.rst

### PR DESCRIPTION
**Pull request by tone-zhang**
Add more information to show the device initialization in SPDK.

Signed-off-by: tone-zhang <tone.zhang@arm.com>

### CODE REVIEW

File: doc/rados/configuration/bluestore-config-ref.rst [hunk 1]
```diff
@@ -301,8 +301,16 @@ set with::
 SPDK Usage
 ==================
 
-If you want to use SPDK driver for NVME SSD, you need to specify NVMe serial
-number here with "spdk:" prefix for ``bluestore_block_path``.
+If you want to use SPDK driver for NVME SSD, at the first some hugepages must
```
Comment (tchaikov) [inline doc/rados/configuration/bluestore-config-ref.rst]: s/at the first/To start with,/

i'd suggest use the active voice when writing document. i.e.

```rst
If you want to use SPDK driver for NVME SSD, you need to ready your system for it. Please refer to `SPDK document`_ for more details.

.. SPDK document: http://www.spdk.io/doc/getting_started.html#getting_started_examples
```

File: doc/rados/configuration/bluestore-config-ref.rst [hunk 2]
```diff
@@ -301,8 +301,16 @@ set with::
...
+be allocated and any NVME device must be unbound from the native kernel
+drivers firstly. SPDK supplies a script to configure the device automatically.
```
Comment (tchaikov) [inline doc/rados/configuration/bluestore-config-ref.rst]: s/firstly//
s/supplies/offers/

File: doc/rados/configuration/bluestore-config-ref.rst [hunk 3]
```diff
@@ -301,8 +301,20 @@ set with::
 SPDK Usage
 ==================
 
-If you want to use SPDK driver for NVME SSD, you need to specify NVMe serial
-number here with "spdk:" prefix for ``bluestore_block_path``.
+If you want to use SPDK driver for NVME SSD, you need to ready your system.
+Please refer to `SPDK document`__ for more detailes.
+
+.. __: http://www.spdk.io/doc/getting_started.html#getting_started_examples
```
Comment (tchaikov) [inline doc/rados/configuration/bluestore-config-ref.rst]: please put
```
.. SPDK document: http://www.spdk.io/doc/getting_started.html#getting_started_examples
```

instead.

Comment (tone-zhang) [reply doc/rados/configuration/bluestore-config-ref.rst]: @tchaikov, Kefu, I passed the build_doc check with the patch. When I changed to ".. SPDK document:",  build_doc reported one error.  Thanks!

**Comment by tchaikov** [comment]
jenkins render docs

File: doc/rados/configuration/bluestore-config-ref.rst [hunk 4]
```diff
@@ -301,8 +301,20 @@ set with::
...
+
+NVME device must be unbound from the native kernel drivers. SPDK offers a
```
Comment (tchaikov) [inline doc/rados/configuration/bluestore-config-ref.rst]: i think this instruction has been covered by http://www.spdk.io/doc/getting_started.html#getting_started_examples .

File: doc/rados/configuration/bluestore-config-ref.rst [hunk 5]
```diff
@@ -301,8 +301,20 @@ set with::
...
+script to configure the device automatically.
+
+For example, users can run the script as root::
+
+  $ sudo src/spdk/scripts/setup.sh
+
+Then ou need to specify NVMe serial number here with "spdk:" prefix for
```
Comment (tchaikov) [inline doc/rados/configuration/bluestore-config-ref.rst]: typo.

**Comment by tone-zhang** [comment]
@tchaikov, Kefu, I updated the patch and removed the redundant description. Could you please have a review? Thanks a lot!

File: doc/rados/configuration/bluestore-config-ref.rst (hunk 4 above)
Comment (tone-zhang) [reply doc/rados/configuration/bluestore-config-ref.rst]: Will update

File: doc/rados/configuration/bluestore-config-ref.rst (hunk 5 above)
Comment (tone-zhang) [reply doc/rados/configuration/bluestore-config-ref.rst]: Will correct it.

**Comment by tone-zhang** [comment]
@tchaikov , Kefu, is there any comment from the latest update? :) 
Could you please have a review? Thanks a lot!

**Comment by tone-zhang** [comment]
@tchaikov ， Kefu， Thanks a lit! :)

**Pull request merged**